vars/mavenNode.groovy

```diff
@@ -5,6 +5,15 @@ def call(Map parameters = [:], body) {
     def defaultLabel = buildId('maven')
     def label = parameters.get('label', defaultLabel)
 
+    nodesDir = "/var/lib/jenkins/nodes"
```

Comment:
could you make it as 

`def nodesDir = "/var/lib/jenkins/nodes"` 

to limit its scope to the local